packages/babel-plugin/src/css-prop/__tests__/index.test.tsx

```diff
@@ -226,7 +291,7 @@ describe('css prop', () => {
     const actual = transform(`
       import '@compiled/core';
 
-      const hello = true;
+      let hello = true;
```

Comment:
these all changed to mutable else they are evaluated to a static representation :)